specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/defenderSettings.json

```diff
@@ -0,0 +1,566 @@
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/defenderSettings/default/downloadManagerActivation": {
+      "post": {
+        "x-ms-examples": {
+          "Download quota data of the subscription": {
+            "$ref": "./examples/DefenderSettings/DownloadManagerActivation.json"
+          }
+        },
+        "tags": [
+          "IoT Defender Settings"
+        ],
+        "operationId": "DefenderSettings_DownloadManagerActivation",
+        "description": "Download manager activation data defined for this subscription",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "produces": [
+          "application/zip"
+        ],
+        "responses": {
+          "200": {
+            "description": "Downloaded file",
+            "schema": {
+              "type": "object",
+              "format": "file"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DefenderSettingsList": {
+      "description": "List of IoT Defender settings",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DefenderSettingsModel"
+          }
+        }
+      }
+    },
+    "DefenderSettingsModel": {
+      "type": "object",
+      "description": "IoT Defender settings",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT Defender settings properties",
+          "$ref": "#/definitions/DefenderSettingsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DefenderSettingsProperties": {
+      "type": "object",
+      "description": "IoT Defender settings properties",
+      "properties": {
+        "deviceQuota": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1000,
+          "description": "Size of the device quota (as a opposed to a Pay as You Go billing model). Value is required to be in multiples of 1000."
+        },
+        "sentinelWorkspaceResourceIds": {
```

Comment:
What do you do with these IDs? What permissions does someone editing defenderSettings need on the workspace IDs they specify here and how do you validate that?

Reply:
Using our first party app, we are resolving the resource ID to get the workspace Id, so we will route data into Sentinel.
At the manifest, we require write permissions over these resource IDs